### PR TITLE
Add support for Rails 6.

### PR DESCRIPTION
**Pull request by joebutler2**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dist: trusty
```
Comment (bricesanchez) [inline .travis.yml]: Why do you use `trusty` ? Does it works with the latest dist?
https://docs.travis-ci.com/user/trusty-to-xenial-migration-guide

File: .travis.yml [hunk 2]
```diff
@@ -9,8 +10,7 @@ notifications:
   webhooks:
     - https://webhooks.gitter.im/e/b5d48907cdc89864b874
 rvm:
-  - 2.5.0
-  - 2.4.1
-  - 2.3.3
-  - 2.2.9
+  - 2.6.1
+  - 2.5.3
+  - 2.4.5
```
Comment (bricesanchez) [inline .travis.yml]: What if we don't specify bugfix version, does it runs against latest minor version?

```suggestion
  - 2.6
  - 2.5
  - 2.4
```

File: .travis.yml (hunk 1 above)
Comment (joebutler2) [reply .travis.yml]: Trusty is the dist that we're using on all of the other refinery gems. It's fixed the build for 2 other projects, and now this one.

Comment (joebutler2) [reply .travis.yml]: I personally have no opinion, what ever fixes the build. :)

File: .travis.yml (hunk 2 above)
Comment (joebutler2) [reply .travis.yml]: Running the build now.

File: .travis.yml [hunk 3]
```diff
@@ -9,8 +10,9 @@ notifications:
   webhooks:
     - https://webhooks.gitter.im/e/b5d48907cdc89864b874
 rvm:
-  - 2.5.0
-  - 2.4.1
-  - 2.3.3
-  - 2.2.9
+  - 2.6.1
+  - 2.5.3
```
Comment (bricesanchez) [inline .travis.yml]: It looks like my code suggestion keep old ruby versions, could you remove 2.6.1 and 2.5.3 ?

**Comment by joebutler2** [comment]
I'm sorry, there is a lot of churn on this PR at the moment. It looks like we need `dist: trusty`, and your suggestion for removing the patch versions worked!

Yes, you are correct, this does not run against Rails 6 in CI. Though this update will enable us to use it in a Rails 6 app. I already have it working on my current project!

**Comment by bricesanchez** [comment]
@parndt i can't merged this PR because it looks like it need `signed commits`. What should we do?

**Comment by parndt** [comment]
Well that's weird, not sure why that was required. Fixed!

**Comment by ttmthanh** [comment]
@parndt  Sorry sir, when I upgrade refinery-authentication to refinery-authentication-devise. I have old data, please tell me "What to do with the database" :(

**Comment by parndt** [comment]
https://github.com/refinery/refinerycms-authentication-devise#installation

**Comment by ttmthanh** [comment]
@parndt Why url changes when we upgrade. /refinery/login --> /refinery/users/login when login false

**Comment by parndt** [comment]
Here's the routes file: https://github.com/refinery/refinerycms-authentication-devise/blob/master/config/routes.rb

It's probably Devise doing that.

**Comment by ttmthanh** [comment]
Can I change the routes as it was originally?

**Comment by ttmthanh** [comment]
Please help me change :'(

**Comment by parndt** [comment]
We haven't changed the routes file in over 4 years. If behaviour has changed, it'll be something to do with Devise. I'd suggest reading through those docs 😄 good luck! I can't be any further help on this.

**Pull request merged**